spec/acceptance/varnish_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'Varnish class' do
+  # Default Varnish 4.1 on CentOS 7 is EOL and drops errors
+  unless fact('os.family') == 'RedHat' && fact('os.release.major') == '7'
+    context 'minimal parameters', 'init.pp' do
+      # Using puppet_apply as a helper
+      # it_behaves_like 'init', 'init.pp'
+      it 'works idempotently with no errors' do
```

Comment:
```suggestion
      it 'works almost idempotently with no errors' do
```

:upside_down_face: 

I see you apply manifest twice before checking for the 3rd run does not make changes, is this avoidable?